src/Agent.Listener/Diagnostics/MtuInfo.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Net.NetworkInformation;
+
+namespace Microsoft.VisualStudio.Services.Agent.Listener.Diagnostics
+{
+    class MtuInfo : IDiagnosticInfo
+    {
+        public void Execute(ITerminal terminal)
+        {
+            NetworkInterface[] nics = NetworkInterface.GetAllNetworkInterfaces();
+            IPGlobalProperties properties = IPGlobalProperties.GetIPGlobalProperties();
+            terminal.WriteLine(string.Format("IP interface information for {0}.{1}", properties.HostName, properties.DomainName));
```

Comment:
can properties be null? maybe properties?.HostName would be safer.

Reply:
Updated since it's an easy change.  In general though I only like adding null checks when null is an expected return clause (it isn't here according to docs).